packages/plugins/users-permissions/admin/src/pages/Providers/utils/forms.js

```diff
@@ -34,7 +34,131 @@ const secretLabel = {
   defaultMessage: 'Client Secret',
 };
 
+const accessToken = {
+  id: getTrad('PopUpForm.Providers.accessToken.label'),
+  defaultMessage: 'Access Token',
+};
+
+const accessTokenSecret = {
+  id: getTrad('PopUpForm.Providers.accessTokenSecret.label'),
+  defaultMessage: 'Access Token Secret',
+};
```

Comment:
I think you will need to add these translations in the en translation file.

Reply:
> I'm a bit confused, why do we need the access token for twitter now? Did something change in their API? Isn't the access token something twitter responds with once the user logs in via SSO? Sorry I haven't had the time to give this a thorough review, but will try to test it later today or tomorrow.

Honestly I haven't checked when it's changed and why. But twitter auth wasn't working for me, I've checked [twitter docs](https://developer.twitter.com/en/docs/authentication/oauth-1-0a) and [purest sources](https://github.com/simov/purest/blob/bc57fc4c8608c24d63f6e05fe85d409949882af8/examples/oauth-1.js#L19) and it seems that all 4 keys are required to make authentication successfully.

Reply:
> I think you will need to add these translations in the en translation file.

@markkaylor  Thanks, added localization